hack/reconciling.yaml

```diff
@@ -66,4 +66,6 @@ resourceTypes:
   - { package: kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1, resourceName: DataVolume, importAlias: cdiv1beta1 }
   
   # velero/v1
-  - {package: github.com/vmware-tanzu/velero/pkg/apis/velero/v1, resourceName: BackupStorageLocation, importAlias: velerov1 }
\ No newline at end of file
+  - { package: github.com/vmware-tanzu/velero/pkg/apis/velero/v1, resourceName: BackupStorageLocation, importAlias: velerov1 }
+
+  - { package: github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2, resourceName: CiliumNetworkPolicy, importAlias: ciliumv2 }
```

Comment:
Nit: add a comment (we have it for other resources)

Reply:
Done. thanks.